src/App.css

```diff
@@ -0,0 +1,129 @@
+*{
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.container {
+  width: 550px;
+  min-width: 550px;
+  background: #23252C no-repeat center fixed;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+
+}
+
+.app {
+  margin-top: 45px;
+  margin-bottom: 45px;
+  background: #1A1B21;
+  border-radius: 10px 10px;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #FFFFFF;
+  max-width: 320px; 
+  
+}
+
+.name {
+  margin-top: 20px;
+  font-weight: 700;
+  font-size: 25px;
+  text-align: center;
+}
+
+.dsgn {
+  font-weight: 400;
+  font-size: 13px;
+  text-align: center;
+  color: #F3BF99;
+  margin-top: 2px;
+  margin-bottom: 8px;  
+}
+
+.website {
+  font-weight: 400;
+  font-size: 10px;
+  text-align: center;
+  color: #F5F5F5;  
+  margin-bottom: 15px;
+}
+
+button {
+  background: #FFFFFF;
+  border: 1px solid #D1D5DB;
+  box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.05);
+  border-radius: 6px;
+  display: flex;
+  gap: 8px;
+  justify-content: center;
+  padding: 9px 11px;
+  max-width: 115px;
+  width: 115px;
+  font-weight: 500;
+  font-size: 14px;
+  color: #374151;
+  cursor: pointer;
+}
+
+.social {
+  display: flex;
+  gap: 17px;
+}
+
+.button-two {
+  background: #5093E2;
+  color: #FFFFFF;
+}
+
+.section-content {
+  max-width: 245px;  
+  font-weight: 400;
+  font-size: 10px;
+  color: #DCDCDC;
+  line-height: 150%; 
+}
+
+.section-header {
+  color: #DCDCDC;
+  font-weight: 700;
+  font-size: 16px;
+  margin-top: 30px;
+  margin-bottom: 5px;
+}
+
+.content-two {
+  margin-bottom: 24px;
+}
+
+.footer {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  background: #161619;
+  padding: 20px;
+  margin-top: 24px;
+  border-radius: 0px 0px 10px 10px;
+}
+
+li {
```

Comment:
Don't use only element tag in CSS file instead please give a class name or use it like .parent_class li { }. It's better to give a class name and every li will have same class.